pkg/apis/monitoring/v1alpha1/scrapeconfig_types.go

```diff
@@ -399,6 +401,8 @@ type HTTPSDConfig struct {
 	// +optional
 	OAuth2         *v1.OAuth2 `json:"oauth2,omitempty"`
 	v1.ProxyConfig `json:",inline"`
+	// +optional
+	v1.CustomHTTPConfig `json:",inline"`
```

Comment:
Why not only HTTP Config?

Reply:
Hmmm... `HTTPConfig`has been defined. @nicolastakashi 
ref: https://github.com/prometheus-operator/prometheus-operator/blob/main/pkg/apis/monitoring/v1/alertmanager_types.go#L461-L490

Reply:
Hmmm 🤔 
Maybe `ScrapeConfigHTTPConfig` naming is hard, but at least this is more concrete, CustomHTTP sounds too generic.

Reply:
Thanks @nicolastakashi 
I got it
Change `CustomHTTPConfig` name to `ScrapeConfigHTTPConfig`

Reply:
hmm HTTP header config isn't limited to scrape resources.

Reply:
I agree, I don't have a better suggestion, just think CustomConfig too broad.